app/src/main/res/values-de/strings-preferences.xml

```diff
@@ -14,6 +14,13 @@
   <string name="pref_title__sub_nav_slider">Navigations Slider</string>
   <string name="pref_desc__sub_nav_slider">Konfiguration der Sichtbarkeit von Einträgen im Navigation-Slider</string>
   <string name="pref_cat__visibility_nav_items">Sichtbarkeit der Einträge</string>
+  <!-- Themes -->
```

Comment:
Do not check in translations (other than `values`). Will get overwritten by Crowdin :)